docs/REMOTE_DEV.md

```diff
@@ -31,6 +31,11 @@ The general remote dev workflow is:
 
 If you forget which stacks you've created, just run `./scripts/happy list` at any time to list the current remote dev stacks.
 
+### Connecting to remote dev databases
+NOTE - You'll need to [install and configure blessclient](https://wiki.czi.team/display/SI/Install+BlessClient) first!
+
+From the root of this repo, run `make remotedb STACK=STACK_NAME_HERE` (replace STACK_NAME_HERE with the name of your stack!) to open a psql console on the remote dev db, or `make remoteconsole STACK=STACK_NAME_HERE` to get a python console connected to the remote dev db.
```

Comment:
oh or actually could we rename to make `remote-pgconsole` and change `remoteconsole` -> `remote-dbconsole` to mirror those target names we have for local?

Reply:
on it!